package.json

```diff
@@ -20,12 +20,13 @@
     "clean:full": "yarn clean && yarn clean:modules",
     "clean:modules": "lerna clean -y && if exist node_modules (rd /s /q node_modules)",
     "efb": "lerna run eslint-fix-build",
-    "publish-sdk": "yarn publish packages/teams-js",
```

Comment:
Are there any release pipelines that will be impacted by this? I think that the plan is to immediately move prod releases to build + release pipeline, so the answer is no, but I want to make sure.

Reply:
No, this isn't used anywhere. There's a few more files that aren't being used either (like npmignore, publish.js). Planning to do a clean up PR once everything is working.